src/storage/src/hummock/sstable/builder.rs

```diff
@@ -200,9 +200,8 @@ impl<W: SstableWriter, F: FilterBuilder> SstableBuilder<W, F> {
         }
         if is_max_epoch(event.new_epoch)
             && self.monotonic_deletes.last().map_or(true, |last| {
-                is_max_epoch(last.new_epoch)
-                    && last.event_key.left_user_key.table_id
-                        == event.event_key.left_user_key.table_id
+                last.event_key.left_user_key.table_id != event.event_key.left_user_key.table_id
+                    || is_max_epoch(last.new_epoch)
```

Comment:
why do we need this change?

Reply:
Because if the left bound of range-tombstone are removed by compaction-filter because of epoch TTL, we can not add the right bound to sstable.
But the epoch of right bound of range-tombstone is MAX, it can not be filtered by compaction-filter

Reply:
Hmm... I still don't get why comparing table id `last.event_key.left_user_key.table_id` and `event.event_key.left_user_key.table_id` is related to epoch TTL. Can you add some documentations?

Reply:
I add. some comment.

Reply:
Thanks for the explanation!